Bukkit/src/main/java/com/plotsquared/bukkit/util/BukkitUtil.java

```diff
@@ -271,23 +271,23 @@ public static BukkitPlayer getPlayer(@NonNull final Player player) {
         if (player == lastPlayer) {
             return lastPlotPlayer;
         }
-        final PlayerManager<?, ?> playerManager = PlotSquared.imp().getPlayerManager();
+        final PlayerManager<?, ?> playerManager = PlotSquared.platform().getPlayerManager();
         return ((BukkitPlayerManager) playerManager).getPlayer(player);
     }
 
     public static Location getLocation(@NonNull final org.bukkit.Location location) {
-        return new Location(location.getWorld().getName(), MathMan.roundInt(location.getX()),
-            MathMan.roundInt(location.getY()), MathMan.roundInt(location.getZ()));
+        return Location.at(com.plotsquared.bukkit.util.BukkitWorld.of(location.getWorld()),
```

Comment:
I'd be tempted to import the P2 BukkitWorld and directly reference the WorldEdit one, as the P2 one is used more.